sources/api/netdog/test_data/net_config/bonding/arpmon_no_targets.toml

```diff
@@ -0,0 +1,10 @@
+[bond1]
+kind = "bond"
+mode = "active-backup"
+interfaces = ["eno53", "eno54"]
+dhcp4 = true
+
+[bond1.monitoring]
+arpmon-interval = 2
+arpmon-validate = "all"
+arpmon-targets = []
```

Comment:
nit: missing newline unless it somehow breaks the unit tests. same for the other TOMLs/XMLs